### ISSUE TITLE
Initial install Accessibility issues for frontpage

### ISSUE THREAD

**Issue by podarok** [ymcatwincities/openy#208]
```
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent|a||Anchor element found with a valid href attribute, but no link content has been supplied.|<a class="navbar-toggle visible-xs visible-sm" data-toggle="collapse" data-target=".sidebar-left" href="#">...</a>
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputSearch.Name|input||This search input element does not have a name available to an accessibility API. Valid names are: element content.|<input type="search" name="query" class="search-input" placeholder="Search"></input>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|a||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.83:1. Recommendation: change background to #007cbd.|<a href="/build8/user/login" data-drupal-link-system-path="user/login">...</a>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|a||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.83:1. Recommendation: change background to #007cbd.|<a href="/build8/openy">...</a>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|a||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.83:1. Recommendation: change background to #007cbd.|<a class="dropdown-toggle" data-toggle="dropdown" href="/build8/openy">...</a>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|a||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.83:1. Recommendation: change background to #007cbd.|<a href="/build8/locations">...</a>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|a||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.83:1. Recommendation: change background to #007cbd.|<a href="/build8/about-ymca">...</a>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|a||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.83:1. Recommendation: change background to #007cbd.|<a href="/build8/give">...</a>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|a||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.83:1. Recommendation: change background to #007cbd.|<a href="/build8/accelerator">...</a>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|a||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.83:1. Recommendation: change background to #007cbd.|<a href="/build8/blog">...</a>
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_1.F77|div|#block-openy-rose-branding|Duplicate id attribute value "block-openy-rose-branding" found on the web page.|<div id="block-openy-rose-branding">...</div>
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.EmptyNoId|a||Anchor element found with no link content and no name and/or ID attribute.|<a class="navbar-toggle" data-toggle="collapse" data-target=".sidebar-left">...</a>
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputSearch.Name|input||This search input element does not have a name available to an accessibility API. Valid names are: element content.|<input type="search" name="query" class="search-input" placeholder="Search"></input>
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_1.F77|nav|#block-openy-rose-user-menu|Duplicate id attribute value "block-openy-rose-user-menu" found on the web page.|<nav role="navigation" aria-labelledby="block-openy-rose-user-menu-menu" id="block-openy-rose-user-menu">...</nav>
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_1.F77|h2|#block-openy-rose-user-menu-menu|Duplicate id attribute value "block-openy-rose-user-menu-menu" found on the web page.|<h2 class="visually-hidden" id="block-openy-rose-user-menu-menu">...</h2>
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_1.F77|nav|#block-openy-rose-main-menu|Duplicate id attribute value "block-openy-rose-main-menu" found on the web page.|<nav role="navigation" aria-labelledby="block-openy-rose-main-menu-menu" id="block-openy-rose-main-menu">...</nav>
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_1.F77|h2|#block-openy-rose-main-menu-menu|Duplicate id attribute value "block-openy-rose-main-menu-menu" found on the web page.|<h2 class="visually-hidden" id="block-openy-rose-main-menu-menu">...</h2>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|p||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.13:1. Recommendation: change background to #008575.|<p>...</p>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|i||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.13:1. Recommendation: change background to #008575.|<i>...</i>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|a||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.53:1. Recommendation: change background to #007faf.|<a href="/build8/user/login?destination=admin/structure/menu%3Ftour%3D1">...</a>
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|a||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.53:1. Recommendation: change background to #007faf.|<a class="button blue" href="/build8/openy?page=1" title="Go to next page" rel="next">...</a>
```

Clean OpenY profile install https://github.com/propeoplemd/yd/pull/3#issuecomment-284369986

**Comment by YMCA-GTC** [comment]
@podarok @Sanchiz Is this still an open issue?

**Comment by podarok** [comment]
Needs to be re-checked

**Comment by podarok** [comment]
Latest front page
```ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|div||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.08:1. Recommendation: change text colour to #fff.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail|h2|#block-useraccountmenu-menu|This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1.08:1. Recommendation: change background to #00192e.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail|h2|#block-mainnavigation-menu|This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1.08:1. Recommendation: change background to #00192e.
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_1.F77|div|#block-openy-rose-branding|Duplicate id attribute value "block-openy-rose-branding" found on the web page.
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.EmptyNoId|a||Anchor element found with no link content and no name and/or ID attribute.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|span||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.86:1. Recommendation: change background to #a26b00.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|span||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.53:1. Recommendation: change background to #007faf.
ERROR|WCAG2AA.Principle1.Guideline1_1.1_1_1.H37|img||Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
ERROR|WCAG2AA.Principle1.Guideline1_1.1_1_1.H37|img||Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|span||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.37:1. Recommendation: change background to #597b8d.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|span||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.13:1. Recommendation: change background to #008575.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|span||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.39:1. Recommendation: change background to #d84009.
```

**Comment by YMCA-GTC** [comment]
Assigned to IXM for resolution

**Comment by paige-kiecker** [comment]
This was resolved in https://github.com/ymcatwincities/openy/pull/1097. Closing out here.
